src/Oceananigans.jl

```diff
@@ -44,8 +44,8 @@ export
     # Coriolis forces
     FPlane, ConstantCartesianCoriolis, BetaPlane, NonTraditionalBetaPlane,
 
-    # BuoyancyModels and equations of state
-    Buoyancy, BuoyancyTracer, SeawaterBuoyancy,
+    # BuoyancyForces and equations of state

```

Comment:
```suggestion
    # BuoyancyFormulations and equations of state
```
?

Reply:
either could work here, though I think what you suggest is better!